importer/celery_tasks.py

```diff
@@ -275,12 +276,16 @@ def create_gn_resource(
 
         hander = import_string(handler_module_path)()
 
-        hander.create_gn_resource(
+        resource = hander.create_gn_resource(
```

Comment:
I would not use abbreviations in API/SPI names, so I suggest to use `create_geonode_resource` here.